src/mscorlib/corefx/System/Globalization/CompareInfo.cs

```diff
@@ -37,6 +37,10 @@ public enum CompareOptions
         OrdinalIgnoreCase = 0x10000000,   // This flag can not be used with other flags.
         StringSort = 0x20000000,   // use string sort method
         Ordinal = 0x40000000,   // This flag can not be used with other flags.
+        
+        // UpperCaseFirst = 0x01000000
```

Comment:
I prefer not to expose this from now as we need to think more if this is the right way to expose it and if we are really have a good story on Windows rather than throwing PNSE.